README.md

```diff
@@ -1,13 +1,18 @@
 # Proposals related to identity credentials
 
-This repo is being used to work on, and hopefully converge, proposals
-related to identity credentials on the web:
+This repo is being used to work on, and hopefully converge on, proposals
+related to the innvocation of digital identity wallets from the web:
 
 * [Identity Credential](identity-credential-proposal.md)
 * [Digital Credential](digital-credential-proposal.md)
 * [navigator.identity.get()](navigator-identity-proposal.md)
 * [Mobile Document Request API](mobile-document-request-api-proposal.md)
 
+## Alternatives being developed elsewhere
+
+* Analysis of [custom URL schemes](custom-schemes.md)
+* [CHAPI](https://chapi.io/) - a browser API [spec](https://w3c-ccg.github.io/credential-handler-api/) and polyfill focusing on wallet selection (rather than credential selection) and enabling both storage and presentation of credentials from both native and web wallets.
+
```

Comment:
@msporny is this a good summary? Also happy to add (or point to) a dedicated page for CHAPI discussing it's advantages / disadvantages relative to the other options we've been discussing here.

Reply:
Yes, good summary, thank you. https://chapi.io/ is the right site to point to.

I'm not sure we need to speak to it's benefits vs. other options as CHAPI's natural progression is (ideally) the Identity Credential API (IC API). If IC API becomes broadly deployed, then CHAPI (eventually) goes away because we got to where we wanted to go -- a native browser API to store/transmit digital credentials). IOW, the ultimate benefits of CHAPI implemented in the browser should align with the benefits of the Identity Credential API implemented in the browser (if we're able to address the remaining sticking points, of which there are few). These "CHAPI benefits" include:

* Direct communication between website and wallet, reducing phishing risk (IC API provides this)
* Reduction in protocol complexity over OID4 (no need for OpenID CSRF protections, PKCE, PIN-approval, HTTP-backend APIs, etc.)
* Query language and protocol agnostic (hopefully IC API goes this route)
* Supports both web-based and native digital wallets (hopefully IC API provides web-based wallet support)
* Keeps UX "in the flow" (if you're on your desktop browser, you don't need to switch to your phone to complete all flows) (IC API provides this)

We are strongly considering upgrading CHAPI to do credential selection if the Identity Credential API goes down this path. Most likely by using the same mechanism as provided by the Identity Credential API (if possible given the polyfill constraints). We want to make sure we get convergence on the same UX and devX -- after considering if there is going to be a backlash from the community over wallet selection vs. credential selection. We might provide credential selection as an (experimental) option that individuals can set: "Don't automatically find credentials for me across wallets [stronger privacy]" vs. "Automatically find credentials for me in my wallet [weaker privacy]" ... and this would be "stronger/weaker privacy" in relation to the polyfill website, which would be able to run code (just like the browser would) to see if a certain wallet had credentials in it.

Reply:
Nice, makes perfect sense. Thanks Manu!